scripts/superdesk-authoring/authoring.js

```diff
@@ -2268,6 +2268,9 @@
                                     });
                             }
                         }
+                        if (config.features.autopopulateByline && !item.byline) {
+                            item.byline = gettext('by') + ' ' + session.identity.display_name;
```

Comment:
it would be better for translators to have it like
```
byline = $interpolate(gettext('by {{ display_name }}')({display_name: session.identity.display_name});
```